src/app/(light)/dashboard/asset/[id]/components/pda-types/meta-data-details.tsx

```diff
@@ -0,0 +1,86 @@
+import CopyData from '@/app/(light)/dashboard/components/copy/copy';
+import CardCell from '@/components/card-cell/card-cell';
+import Tags from '@/components/tags/tags';
+import { DATE_FORMAT } from '@/constants/date';
+import { PrivateDataAsset } from '@/services/server/mock-types';
+import { formatBytes } from '@/utils/bytes';
+import { formatDateDifference } from '@/utils/date';
+
+import { Stack, Divider, Card, Typography } from '@mui/material';
+import dayjs from 'dayjs';
+
+type Props = {
+  pda: PrivateDataAsset;
+};
+
+export default function MetaDataDetails({ pda }: Props) {
+  return (
+    <Stack>
+      <Stack
+        component={Card}
+        variant="outlined"
+        sx={{
+          width: '100%',
+        }}
+      >
+        <Stack divider={<Divider />} sx={{}}>
+          <Stack direction="row" justifyContent="space-between">
+            <CardCell label={'Last Modified'} margin={false} py={3}>
+              <span>{dayjs(pda.updatedAt).format(DATE_FORMAT)}</span>
+            </CardCell>
+            <Divider orientation="vertical" flexItem />
+            <CardCell label={'Expiration'} margin={false} py={3}>
+              <Stack direction={'row'} justifyContent={'space-between'}>
+                <span>{formatDateDifference(pda.expirationDate)}</span>
+                <Typography
+                  variant="caption"
+                  color="text.secondary"
+                  sx={{ mt: 0.3 }}
+                >
+                  {dayjs(pda.expirationDate).format(DATE_FORMAT)}
+                </Typography>
+              </Stack>
```

Comment:
instead of the formatted expiration date (xx/xx/xx hh:mm) have a marginTop, put a alignItems="baseline" on the stack

```suggestion
              <Stack direction={'row'} justifyContent={'space-between'} alignItems="baseline">
                <span>{formatDateDifference(pda.expirationDate)}</span>
                <Typography
                  variant="caption"
                  color="text.secondary"
                >
                  {dayjs(pda.expirationDate).format(DATE_FORMAT)}
                </Typography>
              </Stack>
```